pyro/infer/tracegraph_elbo.py

```diff
@@ -276,9 +272,9 @@ def _loss_and_grads_particle(self, weight, model_trace, guide_trace):
         # have the trace compute all the individual (batch) log pdf terms
         # so that they are available below
         guide_trace.compute_batch_log_pdf(site_filter=lambda name, site: name in guide_vec_md_nodes)
```

Comment:
Sorry, you can also remove `compute_batch_log_pdf()` since that is implied by `.compute_score_parts()`: https://github.com/uber/pyro/blob/dev/pyro/poutine/trace.py#L139

Reply:
can't we remove any more... see latest diff. should probably remove the site filter blah blah from trace soon